src/test/java/edu/ksu/canvas/attendance/submitter/AssignmentValidatorUTest.java

```diff
@@ -136,10 +136,10 @@ public void validateCanvasAssignmentDBMismatchValidationError() throws IOExcepti
     }
 
     @Test
-    public void validateCanvasAssignmentEmptyAssignmentValidationError() throws IOException, AttendanceAssignmentException {
+    public void validateCanvasAssignmentEmptyAssignmentValidationError() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00112)